script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
```

Comment:
Can we either make z optional or query elevation from a public API?

It appears this can be done with gdal http://gis.stackexchange.com/a/29639/3218; @yanliu-chn do have any suggestions?